tracpro/settings/base.py

```diff
@@ -333,7 +333,7 @@ def _org_scheduler_task(task_name):
         'delete',  # can delete an object
         'list',  # can view a list of the objects
     ),
-    'orgs.org': ('create', 'update', 'list', 'edit', 'home'),
+    'orgs.org': ('create', 'update', 'list', 'edit', 'home', 'fetch_runs'),
```

Comment:
So I'm thinking that because the permission is there and it was created during one of my migrations, there might be some issue with smartmin recognizing it when I try to go to that form page as an administrator?